docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Cluster API Reference Use Cases
+creation-date: 2019-04-16
+last-updated: 2019-04-16
+---
+
+# Cluster API Reference Use Cases
+
+This is a living document that serves as a reference and a staging area for use cases collected during post-v1alpha1 project redesign.
+
+## Table of Contents
+
+* [Reference Use Cases](#cluster-api-reference-use-cases)
+  * [Metadata](#metadata)
+  * [Table of Contents](#table-of-contents)
+  * [Role Glossary](#role-glossary)
+  * [Icon Glossary](#icon-glossary)
+  * [Operator of Workload Cluster](#operator-of-workload-cluster)
+  * [Operator of Management Cluster](#operator-of-management-cluster)
+  * [Multi-Cluster/Multi-Provider](#multi-cluster/multi-provider)
+  * [Provider Implementors](#provider-implementors)
+
+## Role Glossary
+- __User__: consumer of a Kubernetes-conformant cluster created by the Cluster API.
+    - Does not use Cluster API.
+- __Operator__: Administrator responsible for creating and managing a Kubernetes cluster deployed by Cluster API.
+    - Uses Cluster API.
+- __Multi-cluster operator__: An operator responsible for multiple Kubernetes clusters deployed by Cluster API.
+    - Uses Cluster API.
+    - Cares about keeping config similar between many clusters.
+
+![Cluster API Use Cases](https://user-images.githubusercontent.com/3118335/56234753-d639e980-603a-11e9-8d45-944c547a3280.png)
+
+## Icon Glossary
+- 🔭 Out of Scope for Cluster API itself, but should be possible via higher level tool.
+    - These are use-cases that we should take care not to prevent.
+
+## Operator of Workload Cluster
+
+### Creating Clusters
+
+- As an operator, given that I have a cluster running Cluster API, I want to be able to use declarative APIs to manage another Kubernetes cluster (create, upgrade, scale, delete).
+
+- As an operator, given that I have a cluster running Cluster API, I want to be able to use declarative APIs to manage a vendor’s Kubernetes conformant cluster (create, upgrade, scale, delete).
```

Comment:
Here just want to confirm: `vendor's Kubernetes` means different Kubernetes distributions, like IBM Cloud Privare, OpenShift, K3S etc, right?

Reply:
maybe reuse the wording in scope and objects
https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/scope-and-objectives.md

External: A control plane offered and controlled by some system other than Cluster API (e.g., GKE, AKS, EKS, IKS).

technically those should be similar to ICP/openshift/K3S?